src/main/java/domain/BettingMoney.java

```diff
@@ -3,14 +3,26 @@
 import java.util.Objects;
 
 public class BettingMoney {
+    private static final Double BLACK_JACK_BONUS = 1.5;
+    private static final Integer LOSE_MONEY = -1;
 
-    private final int money;
+    private int money;
 
     public BettingMoney(int money) {
         this.money = money;
     }
 
-    public int getBettingMoney(){
+    public int bust() {
+        this.money = this.money * LOSE_MONEY;
+        return this.money;
+    }
```

Comment:
bust() 메소드의 return 값이 int 인 부분이 어색해보입니다.

1. 1.5배, -1 배를 계산하는 로직이 bettingMoney의 책임이 맞는 지 고민해보면 좋을 것 같고,
2. 맞다고 판단한다면 메소드 네이밍을 수정해야하지 않을까 생각합니다.

+ 유지한다면 this.money의 final 은 유지시키고 아래와 같이 진행해야한다 생각합니다.
```suggestion
    public int bust() {
        return this.money * LOSE_MONEY;
    }
```

Reply:
bettingMoney의 책임이 아닌 것 같아서 Result enum타입에서 수익 배율을 계산하도록 하였습니다.